src/main/java/com/openmrs/migrator/core/utilities/ConsoleUtils.java

```diff
@@ -0,0 +1,117 @@
+package com.openmrs.migrator.core.utilities;
+
+import java.io.Console;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class ConsoleUtils {
+
+  public static Optional<String> getDatabaseDetaName(Console console) {
+
+    checkConsoleAvailability(console);
+    Optional<String> wrappedName = Optional.empty();
+
+    console.writer().println("Database name:");
+    String dataBaseName = console.readLine();
+
+    wrappedName = Optional.of(dataBaseName);
+
+    return wrappedName;
+  }
+
+  private static void checkConsoleAvailability(Console console) {
+
+    if (console == null) {
+      throw new IllegalArgumentException("System.console() can't be  null");
+    }
+  }
+
+  public static boolean isConnectionIsToBeStored(Console console) {
+    checkConsoleAvailability(console);
+    console.writer().println("Do you want to keep this connection? [y/n]");
+    String answer = console.readLine();
+    return "y".equals(answer);
+  }
+
+  public static int startMigrationAproach(Console console) {
```

Comment:
I think lets simplify this a bit - too many options makes it hard to chose for the user.

Lets limit to only:
Get database names
1. Provide database name for existing db in MySQL server
2. Use databases names from settings file

Location of databases:
1. Already loaded in MySQL
2. Use sql dump files

Reply:
done

Reply:
hi dear colleagues, we now load settings.properties into kettle environment directly without having to touch ~/.kettle/kettle.properties at all. this implies we only need an option to set the dabase name and load backups and my suggestion is moving these into setup command so that run command just triggers runAllJobs directly without any user choices.